index.js

```diff
@@ -27,13 +28,21 @@ const {
 
 const app = express();
 const port = PORT || 3000;
+const sess = {
+  secret: GRAVITY_SECRET,
```

Comment:
It would just be an ENV var set, `METAPHYSICS_SESSION_SECRET` or whatever, so not a ton of overhead, but I'm not opinionated enough. This is fine/awesome.

👍